galasa-managers-parent/galasa-managers-testingtools-parent/dev.galasa.jmeter.manager/src/main/java/dev/galasa/jmeter/IJMeterSession.java

```diff
@@ -60,7 +60,7 @@ public interface IJMeterSession {
      * JMX-files have to be prepared for this by replacing ALL the $P__VARIABLE notations with $VARIABLE notations
      * Then by adding neccesary parameters inside a HashMap with it, it is possible to change the target of a test dynamically inside the test
      * Example:
```

Comment:
Nope. Use this:
/**
* <PRE>
HashMap<String,Object> map = new HashMap<String,Object>();
map.put("HOST", "galasa.dev");
session.setChangedParametersJmxFile(jmxStream, map);
</PRE>
* @param jmxStream
*/